pipeline/filters/kubernetes.md

```diff
@@ -46,6 +46,8 @@ The plugin supports the following configuration parameters:
 | Dummy\_Meta | If set, use dummy-meta data \(for test/dev purposes\) | Off |
 | DNS\_Retries | DNS lookup retries N times until the network start working | 6 |
 | DNS\_Wait\_Time | DNS lookup interval between network status checks | 30 |
+| Use\_Kubelet | this is an optional feature flag to get metadata information from kubelet instead of calling Kube Server API to enhance the log. This could mitigate the Kube API heavy traffic issue for large cluster. This feature default to be disabled and user could enable it by set this flag to true | False |
```

Comment:
the option supports `On` or `Off` as values as well right? (it should, boolean options in fluent bit config maps support that). I think it's preferable in the docs to use Off and On instead of true and false, even though both work. You can see this for other options.

Reply:
Also, lets change the wording here, this is my suggestion:

> This is an optional feature flag to get metadata information from kubelet instead of calling Kube Server API to obtain kubernetes metadata. At large scale, [issues have been reported](https://github.com/fluent/fluent-bit/issues/1948) with Fluent Bit making calls to the API Server, this option may mitigate those problems. 

We can link to the issue instead of saying somethig vague, folks won't know what the "Kube API heavy traffic issue" means without more information.

Reply:
Yeah it supports `On`  and `Off`. Changed the default value to `Off` and updated the description.